packages/eds-core-react/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -52,26 +52,28 @@ export default {
   },
 } as ComponentMeta<typeof Accordion>
 
-export const Introduction: Story<AccordionProps> = (args) => (
-  <Accordion {...args}>
-    <Accordion.Item isExpanded>
-      <Accordion.Header>Header 1</Accordion.Header>
-      <Accordion.Panel>Content 1</Accordion.Panel>
-    </Accordion.Item>
-    <Accordion.Item>
-      <Accordion.Header>Header 2</Accordion.Header>
-      <Accordion.Panel>Content 2</Accordion.Panel>
-    </Accordion.Item>
-    <Accordion.Item>
-      <Accordion.Header>Header 3</Accordion.Header>
-      <Accordion.Panel>Content 3</Accordion.Panel>
-    </Accordion.Item>
-  </Accordion>
-)
+export const Introduction: Story<AccordionProps> = (args) => {
+  return (
+    <Accordion {...args}>
+      <Accordion.Item isExpanded>
+        <Accordion.Header>Header 1</Accordion.Header>
+        <Accordion.Panel>Content 1</Accordion.Panel>
+      </Accordion.Item>
+      <Accordion.Item>
+        <Accordion.Header>Header 2</Accordion.Header>
+        <Accordion.Panel>Content 2</Accordion.Panel>
+      </Accordion.Item>
+      <Accordion.Item>
+        <Accordion.Header>Header 3</Accordion.Header>
+        <Accordion.Panel>Content 3</Accordion.Panel>
+      </Accordion.Item>
+    </Accordion>
+  )
+}
 
 export const Header: Story<AccordionProps> = () => {
   return (
-    <Wrapper>
+    <>
```

Comment:
Just a heads up, in React, if a single element is return you don't need fragments. Fragments is only used when you have multiple elements returned.